### DISCUSSION TITLE
UCG Ultra

### DISCUSSION THREAD

**Discussion by viotemp1** [unifi-utilities/unifi-common#626] [Q&A] [answered]
Will this scripts work on UCG Ultra. I see that folders are not same like UDR/UDM.
Viorel

**Comment by boostchicken** [comment]
Only one way to find out and thats try them.  

What are these folders that are different?  Have examples?

**Reply by viotemp1** [reply]
these are the mounts:
Filesystem                         Size  Used Avail Use% Mounted on
udev                               1.5G     0  1.5G   0% /dev
tmpfs                              296M   37M  259M  13% /run
/dev/disk/by-partlabel/root        2.0G  1.2G  685M  63% /boot/firmware
/dev/loop0                         571M  571M     0 100% /mnt/.rofs
/dev/disk/by-partlabel/overlay     9.3G  1.6G  7.2G  18% /mnt/.rwfs
overlayfs-root                     9.3G  1.6G  7.2G  18% /
/dev/disk/by-partlabel/log         974M   93M  815M  11% /var/log
/dev/disk/by-partlabel/persistent  2.0G  123M  1.7G   7% /persistent
tmpfs                              1.5G  224K  1.5G   1% /dev/shm
tmpfs                              5.0M     0  5.0M   0% /run/lock
tmpfs                              739M  332K  738M   1% /tmp
tmpfs                               16M  9.0M  7.1M  56% /var/log/ulog
tmpfs                               64M     0   64M   0% /var/opt/unifi/tmp
I guess only persistent will survive reboot, I did not try yet.

**Reply by boostchicken** [reply]
Wow Ubiquiti, we have "persistent" again.   Just making changes that have no benefit and being consitutionally incapable of defining and adopting a pattern.

**Comment by bslatyer** [comment, marked as answer]
Fixed by #625
